osu.Game.Mode.Osu/Objects/Drawables/Pieces/SliderBody.cs

```diff
@@ -88,6 +81,45 @@ private void load(OsuConfigManager config)
         {
             snakingIn = config.GetBindable<bool>(OsuConfig.SnakingInSliders);
             snakingOut = config.GetBindable<bool>(OsuConfig.SnakingOutSliders);
+
+            int textureWidth = (int)PathWidth * 2;

```

Comment:
Needs to be multiplied by the maximum of the x and y components of DrawInfo.Matrix.ExtractScale